model/build/db.go

```diff
@@ -43,6 +43,8 @@ var (
 	AbortedKey             = bsonutil.MustHaveTag(Build{}, "Aborted")
 
 	TaskCacheIdKey = bsonutil.MustHaveTag(TaskCache{}, "Id")
+
+	BlockedKey = "Blocked"
```

Comment:
We should use bsonutil.MustHaveTag like above (also should be up with the other Build tags)

Reply:
done